Source/WebCore/page/SecurityOrigin.cpp

```diff
@@ -596,13 +598,17 @@ Ref<SecurityOrigin> SecurityOrigin::create(const String& protocol, const String&
     return origin;
 }
 
-Ref<SecurityOrigin> SecurityOrigin::create(WebCore::SecurityOriginData&& data, String&& domain, String&& filePath, Markable<OpaqueOriginIdentifier, OpaqueOriginIdentifier::MarkableTraits>&& opaqueOriginIdentifier, bool universalAccess, bool domainWasSetInDOM, bool canLoadLocalResources, bool enforcesFilePathSeparation, bool needsStorageAccessFromFileURLsQuirk, std::optional<bool> isPotentiallyTrustworthy, bool isLocal)
+Ref<SecurityOrigin> SecurityOrigin::create(const SecurityOriginData& data)
```

Comment:
Would we be able to use `SecurityOriginData&&` and `WTFMove(data)` below? I found one call site (not sure if there are more) and this call site seems to pass an isolated copy so it would benefit from using a r-value reference.